packages/contentstack-utilities/src/helpers.ts

```diff
@@ -8,3 +9,20 @@ export const doesBranchExist = async (stack, branchName) => {
       return error;
     });
 };
+
+export const isManagementTokenValid = async (stackAPIKey, managementToken) => {
+  const httpClient = new HttpClient({ headers: { api_key: stackAPIKey, authorization: managementToken } });
+  try {
+    const response = (await httpClient.get(`${configHandler.get('region').cma}/v3/environments?limit=1`))?.data;
+
+    if (response?.environments) {
+      return { valid: true, message: `valid token and stack api key` }
+    } else if(response?.error_code) {
+      return { valid: false, message: response.error_message };
+    } else {
+      throw typeof response === "string"? response : "";
```

Comment:
@cs-raj if response is not a string, it throws an empty string, can we keep some default message here so that user wont be confused

Reply:
@shafeeqd959 when the response would not be string we are having default message as  **message:`Failed to check the validity of the Management token.**.